src/u2f.h

```diff
@@ -15,17 +15,21 @@
 #ifndef _U2F_DEVICE_H_
 #define _U2F_DEVICE_H_
 
-#include "usb/u2f/u2f.h"
-#include "usb/usb.h"
 #include <stdbool.h>
```

Comment:
remove all includes here, none are used by the header

Reply:
done